doc/en/reference.rst

```diff
@@ -161,6 +161,20 @@ Skip a test function if a condition is ``True``.
     :keyword str reason: Reason why the test function is being skipped.
 
 
+.. _`pytest.mark.usefixtures ref`:
+
+pytest.mark.usefixtures
+~~~~~~~~~~~~~~~~~~~~~~~
+
+**Tutorial**: :ref:`usefixtures`.
+
+Mark a test function as using the given fixture names.
```

Comment:
I wonder if it could be a good idea to add an explicit warning/reminder here that you can only mark test functions like this and not other fixtures to address https://github.com/pytest-dev/pytest/issues/1014

Reply:
Ahh good idea, will do

Reply:
Something like:

"Be aware that although you can use fixtures from oher fixtures by adding them to the function parameters, you cannot do with by marking the fixture function"

Maybe even with an example?

This works:

```
@pytest.fixture
def my_fixture(my_other_fixture):
    [...]
```

This will **not** yield the wanted result and currently not result in an error or a warning either.

```
@pytest.mark.usefixtures("my_other_fixture")
@pytest.fixture
def my_fixture_that_sadly_wont_use_my_other_fixture():
    [...]
```

I would even suggest to put this in a warning box, because this can be surprising even for experienced pytest users, especially due to the lack of an error atm.

Reply:
Thanks @obestwalter!

I added the warning but the example I added to the tutorial/normal documentation. IMHO we should keep the reference as lean as possible.